.github/workflows/create-release-branch.yml

```diff
@@ -66,27 +68,69 @@ jobs:
           else
             RELEASE_TYPE=minor
           fi
-          # Increment the version as needed
+          # Increment the version as needed.
           npm --no-git-tag-version version $RELEASE_TYPE
-          git add package.json
-      # Update the dependencies
-      - name: Update dependencies
-        run: |
-          yarn upgrade
-          node tools/UpdateRetireJsVulns.js
-          git add yarn.lock
-          git add retire-js
-      # Create the new branch
-      - name: Checkout new branch, commit, push
-        id: create-branch
+      # Create the new branch as a direct copy of `dev` and push it to GitHub so the API can
+      # interact with it later.
+      - id: create-branch
         run: |
           NEW_VERSION=$(jq -r ".version" package.json)
+          git stash
+          git checkout dev
+          git pull
           git checkout -b release-$NEW_VERSION
-          git config --global user.name "sfca-bot"
-          git config --global user.email "cli-scanner@salesforce.com"
-          git commit -m "Incrementing version for $NEW_VERSION release"
           git push --set-upstream origin release-$NEW_VERSION
```

Comment:
We now create the branch as a direct copy of `dev`, because the API call need an existing branch to push the new commit to. It can't create the branch from scratch.